.gitignore

```diff
@@ -1,3 +1,6 @@
+# Project
+deleteme
```

Comment:
??? delete me ??? who is going to delete it? 
Why, sometimes I've believed as many as six impossible things before breakfast.

Reply:
removed, was a test